src/manifolds/MetricManifold.jl

```diff
@@ -51,6 +51,126 @@ inner product ``g(X, X) > 0`` whenever ``X`` is not the zero vector.
 """
 abstract type RiemannianMetric <: AbstractMetric end
 
+@doc raw"""
+    change_gradient(M::AbstractManifold, G2::AbstractMetric, p, X)
+
+Convert the gradint `X` at `p` on the[`AbstractManifold`](@ref) `M` from one metric to another.
+
+Assume that for a real-valued function ``f: \mathcal M \to ℝ`` we are given the Riesz representer of the differential with respect to the metric ``g_2`` i.e.
+
+```math
+    g_2(X,Y) = Df(p)[Y] \quad \text{for all } Y ∈ T_p\mathcal M.
+```
+
+In order to convert this into the gradient with respect to the (implicitly given) metric ``g_1`` of `M`,
+we have to find the conversion function ``c: T_p\mathcal M \to \T_p\mathcal M`` such that
+
+```math
+    g_2(X,Y) = g_1(c(X),Y)
+```
+
+If both metrics are given in their [`local_metric`](@ref) (symmetric positive defintie) matrix
+representations ``G_1`` and ``G_2`` and ``x,y`` are the local coordinates with respect to
+the same basis of the tangent space, the equation reads
+
+```math
+   x^*G_2y = c(x)^*G_1 y \quad \text{for all } y \in ℝ^d,
+```
+where `\cdot^*`` denotes the conjugate transpose.
+
+and we obtain `c(x) = (G_1\backslask G_2)^*x `
+
+# Examples
+
+    change_gradient(Sphere(2), EuclideanMetric(), p, X)
+
+Since the metric in ``T_p\mathbb S^2`` is the Euclidean metric from the embedding restricted to ``T_p\mathbb S^2``, this just returns `X`
+
+    change_gradient(SymmetricPOsitiveDefinite(3), EuclideanMetric(), p, X)
+
+Here, the default metric in `\mathcal P(3)` is the [`LinearAffineMetric`](@ref) and the transformation can be computed as ``pXp``
+"""
+change_tangent(::AbstractManifold, ::AbstractMetric, ::Any, ::Any)
```

Comment:
```suggestion
change_gradient(::AbstractManifold, ::AbstractMetric, ::Any, ::Any)
```
or whatever else we end up calling this :slightly_smiling_face: